CONTRIBUTING.md

```diff
@@ -36,10 +54,9 @@ clang-format -style=file -i         \
      libsyclinterface/helper/source/*.cpp
 ```
 
-> **_NOTE:_** A much simpler option is to use `pre-commit` and the
-> `clang-format` hook that we provide.
+> **NOTE:** It is easier to use `pre-commit` and the `clang-format` hook.
 
-### Python code style
+### Python Code Style
 
 We use the following Python code style tools:
 - [black](https://black.readthedocs.io/en/stable/) code formatter.
```

Comment:
I prefer we remove the versions of the tools from the contributing.md and instead ask people to look at the [.pre-commit-config.yaml](https://github.com/alexandraepan/dpctl/blob/infodev-doc-review/.pre-commit-config.yaml) file. We will almost always forget to keep the contributing.md updated every time we update the the pre-commit-config.yaml.

Reply:
agreed

Reply:
done